CMakeLists.txt

```diff
@@ -173,6 +178,47 @@ if(MSVC)
     # Library needs /EHsc (Enable C++ exceptions)
 endif()
 
+#--- Package
+include(CMakePackageConfigHelpers)
+
+write_basic_package_version_file(
+  ${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}-config-version.cmake
+  VERSION ${DIRECTXTK_VERSION}
+  COMPATIBILITY AnyNewerVersion)
+
+install(TARGETS ${PROJECT_NAME}
+  EXPORT ${PROJECT_NAME}-targets
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION bin)
+
+configure_package_config_file(${CMAKE_CURRENT_SOURCE_DIR}/.nuget/${PROJECT_NAME}-config.cmake.in
+  ${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}-config.cmake
+  INSTALL_DESTINATION cmake/})
+
+install(EXPORT ${PROJECT_NAME}-targets
+  FILE ${PROJECT_NAME}-targets.cmake
+  NAMESPACE Microsoft::
+  DESTINATION cmake/)
+
+if((BUILD_XAUDIO_WIN10) OR (BUILD_XAUDIO_WIN8))
```

Comment:
I'm sure there's some cmake magic syntax I could use here to make this a bit more compact, but I'm not sure what it is...

I want to always exclude XboxDDSTextureLoader.h from the installed headers (CMake is only building the PC versions), but I also want to exclude Audio.h if I'm not building audio support.